kubernetes/check.py

```diff
@@ -34,6 +34,8 @@
     'cpu.*.total']
 DEFAULT_COLLECT_EVENTS = False
 DEFAULT_NAMESPACES = ['default']
+DEFAULT_SEND_CPU_AS_CORES = False
```

Comment:
I think it's a good idea to have it disabled by default, but I don't see it used?